internal/host/monitor_test.go

```diff
@@ -58,6 +58,8 @@ var _ = Describe("monitor_disconnection", func() {
 			{Status: api.Success, ValidationId: string(models.HostValidationIDOcsRequirementsSatisfied)},
 			{Status: api.Success, ValidationId: string(models.HostValidationIDLsoRequirementsSatisfied)},
 		}, nil)
+		mockOperators.EXPECT().GetCPURequirementForRole(gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
The return instruction for the mock is missing - how will the mock behave in this case?

Reply:
I checked with the debugger: it returns the default values for each type: `0` and `nil`. In this case I think those values are valid since the test does not focus on the CPU and memory validation for the host.